desktop/ui/src/main/java/org/datacleaner/widgets/Dropzone.java

```diff
@@ -115,8 +115,13 @@ protected void showFileChooser() {
             final File file = fileChooser.getSelectedFile();
 
             if (file.exists()) {
-                final Datastore datastore = DatastoreCreationUtil.createAndAddUniqueDatastoreFromFile(
-                        _datastoreCatalog, file);
+                final Datastore datastore;
+                final String filename = file.getName();
+                if (_datastoreCatalog.containsDatastore(filename)) {
```

Comment:
This checks if the name of the datastore is same as the filename. What if I have this file in my datastores already, but I gave it a custom name instead of a default filename-based name? Shouldn't we check for the whole path to make sure it is the same file?